kitchen/bin/image-tagging.json

```diff
@@ -0,0 +1,13 @@
+{
+  "cmd": "COMMAND --port ${PORT0}",
+  "concurrency-level": 1,
+  "cpus": 1,
+  "health-check-url": "/status",
+  "idle-timeout-mins": 10,
+  "max-instances": 20,
+  "mem": 1024,
+  "name": "image-tagging-service",
+  "run-as-user": "RUN_AS_USER",
+  "token": "image-tagging",
+  "version": "0.1.0-SNAPSHOT"
+}
```

Comment:
Can we put this file under /demos/tensorflow-image-tagging instead of under /kitchen?

Reply:
Done.